scripts/sct_dmri_moco.py

```diff
@@ -40,78 +41,109 @@ def get_parser():
-                      mandatory=False,
-                      default_value='1',
-                      example=['0', '1'])
-    parser.add_option(name="-v",
-                      type_value='multiple_choice',
-                      description="verbose: 0 = nothing, 1 = classic, 2 = expended",
-                      mandatory=False,
-                      example=['0', '1', '2'],
-                      default_value='1')
+    parser = argparse.ArgumentParser(
+        description=(
+            '  Motion correction of dMRI data. Some of the features to improve robustness were proposed in Xu et al. '
+            '(http://dx.doi.org/10.1016/j.neuroimage.2012.11.014) and include:\n'
+            '    -group-wise (-g)\n'
+            '    -slice-wise regularized along z using polynomial function (-param). For more info about the method, '
+            'type: isct_antsSliceRegularizedRegistration\n'
+            '    -masking (-m)\n'
+            '    -iterative averaging of target volume\n'
+        ),
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help="Diffusion data. Example: dmri.nii.gz"
+    )
+    mandatory.add_argument(
+        '-bvec',
+        metavar=Metavar.file,
+        required=True,
+        help='Bvecs file. Example: bvecs.txt'
+    )
+
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="Show this help message and exit."
+    )
+    optional.add_argument(
+        '-bval',
+        metavar=Metavar.file,
+        default=param_default.fname_bvals,
+        help='Bvals file. Example: bvals.nii.gz',
+    )
+    optional.add_argument(
+        '-bvalmin',
+        type=float,
+        metavar=Metavar.float,
+        default=param_default.bval_min,
+        help='B-value threshold (in s/mm2) below which data is considered as b=0. Example: 50.0',
+    )
+    optional.add_argument(
+        '-g',
+        type=int,
+        metavar=Metavar.int,
+        default=param_default.group_size,
+        help='Group nvols successive dMRI volumes for more robustness. Example: 2',
+    )
+    optional.add_argument(
+        '-m',
+        metavar=Metavar.file,
+        default=param_default.fname_mask,
+        help='Binary mask to limit voxels considered by the registration metric. Example: dmri_mask.nii.gz',
+    )
+    optional.add_argument(
+        '-param',
+        metavar=Metavar.list,
+        type=list_type(',', str),
+        help=f"R|Advanced parameters. Assign value with \"=\", and separate arguments with \",\".\n"
+             f"    -poly [int]: Degree of polynomial function used for regularization along Z. For no regularization set to "
```

Comment:
(same for the other items)
```suggestion
             f"  - poly [int]: Degree of polynomial function used for regularization along Z. For no regularization set to "
```